cvelib/cve_api.py

```diff
@@ -87,6 +90,14 @@ def post(self, path, **kwargs):
     def put(self, path, **kwargs):
         return self.http_request("put", path, **kwargs)
 
+    def create(self, cve_id, cve_json):
+        """Create a CVE from a JSON object representing the CNA container data.
+        Only the "affected", "descriptions", "providerMetadata", and "references" properties may be submitted.
+        See https://github.com/CVEProject/cve-services/issues/584 for details."""
```

Comment:
```suggestion
        """Create a CVE record from a JSON object representing the CNA container data.

        Only the "affected", "descriptions", "providerMetadata", and "references" properties may be
        submitted. See https://github.com/CVEProject/cve-services/issues/584 for details.
        """
```

I assume this refers to the fact that you only need to submit the CNA container itself, right? That container has a ton of other attributes apart from the four listed above though. Does that mean CVE Services 2.1.0 does not accept e.g. "workarounds" as an attribute within a CNA container?

Reply:
No, this was due to a confusing error message, "Additional properties are not supported", when trying to submit the four required properties mentioned here plus a "title" property. I thought it meant that only required properties are allowed, and optional properties are not accepted.

But per discussion in that Github issue, it's actually a bug with the current JSON schema being out-of-date. The "title" property is supposed to appear inside the "providerMetadata" key, not as a top-level key.

I removed this comment, and added a link to the JSON schema used by CVE Services in the docs, so that users know what data is allowed. And hopefully they will update the JSON schema to the latest version before / as part of the the 2.1.0 release.